### PR TITLE
Add a new action with customized vibration

### PR DESCRIPTION
**Pull request by Bouh**
Ready to merge

### CODE REVIEW

File: docs/gdevelop5/all-features/gamepad.md [hunk 1]
```diff
@@ -64,10 +64,13 @@ You can then use it to set a force on an object (or anything else that you want
...
 
 ![](/gdevelop5/all-features/gamepad-condition-connected.png)
 
 ## Vibrate a gamepad
 
-Use the action *Gamepad vibration* to make a gamepad to vibrate.
\ No newline at end of file
+Use the action **Gamepad vibration** to make a gamepad to vibrate.
+
+**Advanced Gamepad vibration** allow you to setup the magnitude of the vibrations with two new values, the low and high value for the motors rumble.
```
Comment (D8H) [inline docs/gdevelop5/all-features/gamepad.md:75]:
```suggestion
**Advanced Gamepad vibration** allows to setup the magnitude of the vibrations with a lower and a higher bound for motors rumble.
```

**Review by D8H — APPROVED** [review]

File: docs/gdevelop5/all-features/gamepad.md [hunk 2]
```diff
@@ -64,10 +64,13 @@ You can then use it to set a force on an object (or anything else that you want
...
 
 ![](/gdevelop5/all-features/gamepad-condition-connected.png)
 
 ## Vibrate a gamepad
 
-Use the action *Gamepad vibration* to make a gamepad to vibrate.
\ No newline at end of file
+Use the action **Gamepad vibration** to make a gamepad vibrate.
+
+**Advanced Gamepad vibration** allows to setup the magnitude of the vibrations with a lower and a higher bound for motors rumble.
```
Comment (D8H) [inline docs/gdevelop5/all-features/gamepad.md:75]: Sorry, actually, I'm not sure about the change I suggested.
What are the low and high values? Is it 2 motors: one for small high frequency vibrations and one for big low frequency ones?

Comment (Bouh) [reply docs/gdevelop5/all-features/gamepad.md:75]: From the [w3c spec](https://w3c.github.io/gamepad/extensions.html#gamepadhapticeffecttype-enum):

> [strongMagnitude](https://w3c.github.io/gamepad/extensions.html#dom-gamepadeffectparameters-strongmagnitude) and [weakMagnitude](https://w3c.github.io/gamepad/extensions.html#dom-gamepadeffectparameters-weakmagnitude) set the vibration intensity levels for the low-frequency and high-frequency rumble motors, normalized to the range [0,1], default 0. An effect with magnitude values outside of this range is not a [valid effect](https://w3c.github.io/gamepad/extensions.html#dfn-valid-effect).

Comment (D8H) [inline docs/gdevelop5/all-features/gamepad.md:75]:
```suggestion
**Advanced Gamepad vibration** allows to setup vibration magnitudes for low and high frequency rumble motors.
```

Comment (Bouh) [reply docs/gdevelop5/all-features/gamepad.md:75]:
```suggestion
**Advanced Gamepad vibration** allows to setup the strong magnitude and weak magnitude set the vibration intensity levels for the low-frequency and high-frequency rumble motors.
``````

**Pull request merged**